### PR TITLE
Update harisdev-netizen.json

### PR DESCRIPTION
**Pull request by harisdev-netizen**
Added a new Links to my profile kindly check into it. Thank you!

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/harisdev-netizen.json [hunk 1]
```diff
@@ -37,6 +37,11 @@
       "url": "https://www.linkedin.com/in/hariskhan00/",
       "icon": "FaLinkedin"
     }
```
Comment (sital002) [inline data/harisdev-netizen.json:39]:
```suggestion
    },
```
There is comma missing that's why the tests are failing.

**Review by sital002 — CHANGES_REQUESTED** [review]

File: data/harisdev-netizen.json [hunk 2]
```diff
@@ -36,7 +36,12 @@
       "name": "LinkedIn: Connect with me",
       "url": "https://www.linkedin.com/in/hariskhan00/",
       "icon": "FaLinkedin"
-    }
+    },
+    {
+      "name": "instantBazar: My Online Store",
+      "url": "https://www.instagram.com/instantbazar_/",
+      "icon": "FaInstagram"
+    },
```
Comment (SaraJaoude) [inline data/harisdev-netizen.json:44]: delete comma:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by harisdev-netizen** [comment]
> looks good

Hey there @SaraJaoude my changes had not been yet merged. It isn't yet displayed on my BioDrop profile kindly look into it. And also guide me on how to latest testimonials to my profile through JSON. I think I have already created a PR if you can check that out as well

**Pull request merged**